### PR TITLE
Registrationform

### PR DESCRIPTION
**Pull request by phanit1**
Completed Today Task Bro

### CODE REVIEW

File: templates/registrationdata.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html>
+   <body style="font-size: large;">
+    Registration successful<br>
+    Registered user email id is below:
+      <table border = 1>
+         {% for key, value in registrationdata.items() %}
```
Comment (ashishk144) [inline templates/registrationdata.html:7]: instead of running a loop, you could directly access the email with the key.

File: templates/reg.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```
Comment (ashishk144) [inline templates/reg.html:7]: Is jquery required?

Comment (phanit1) [reply templates/reg.html:7]: Scripts are required as it is required for responsiveness i.e.,when using website in mobile view

File: templates/registrationdata.html (hunk 1 above)
Comment (phanit1) [reply templates/registrationdata.html:7]: I thought of doing like this.
If change is required i will try to change 
If not I will not change it and keep it as before

Comment (ashishk144) [reply templates/registrationdata.html:7]: Running a loop is time consuming, it's better to get a direct access.

File: templates/reg.html (hunk 2 above)
Comment (ashishk144) [reply templates/reg.html:7]: okay.

File: templates/registrationdata.html (hunk 1 above)
Comment (phanit1) [reply templates/registrationdata.html:7]: k bro i will try it. Can i start todays assignment

Comment (ashishk144) [reply templates/registrationdata.html:7]: yes, you can

File: reg.py [hunk 3]
```diff
@@ -8,6 +8,7 @@ def hello_world():
 def hello():
     if request.method == 'POST':
       data = request.form
+      print("Email Id:"+data["email"])
```
Comment (ashishk144) [inline reg.py:11]: are you using this file, if not, delete them.

File: create.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        print("Registration Success")
+        var1 ='Registration Success'
+        return render_template("reg.html", var1 = var1)
+    else:
+        return render_template("reg.html")
+    
+
+@app.route('/admin')
+def admin():
+    data = USERS.query.all()
```
Comment (ashishk144) [inline create.py]: Users data needs to be printed in the descending order of timestamp.

**Review by ashishk144 — COMMENTED** [review]
Move create.py to application.py and remove the create.py file

File: create.py (hunk 4 above)
Comment (phanit1) [reply create.py]: Updated bro please check once so that I can start login part

File: reg.py (hunk 3 above)
Comment (phanit1) [reply reg.py:11]: just kept if it is required afterwards

File: create.py (hunk 4 above)
Comment (ashishk144) [reply create.py]: looks good

**Comment by ashishk144** [comment]
Move the contents of create.py to application.py and then you're good to merge.

File: create.py (hunk 4 above)
Comment (phanit1) [reply create.py]: shall i move the  contents of create.py to application.py
I kept it in another because i thought it may reflect to any merge conflicts

**Pull request merged**